lib/rules/web/admin_console/4.3/common_ui_elements.xyaml

```diff
@@ -121,6 +121,11 @@ click_kebab_of_one_resource:
 click_one_operation_in_kebab:
   action: click_kebab_of_one_resource
   action: click_button
+check_field_on_resource_list_page:
+  elements:
+  - selector:
+      xpath: //th[contains(@data-label,'<field>')]
+    timeout: 30
```

Comment:
Are we waiting table loaded completely here? If yes, I would say removing timeout from this action and use another step `wait_table_loaded` in feature, that looks much meaningful